tests/functional/aws-node-sdk/lib/utility/website-util.js

```diff
@@ -216,6 +217,25 @@ function _assertResponseHtmlRedirect(response, type, redirectUrl, method) {
         'Best redirect link ever');
         _assertResponseHtml(response.body, 'h1',
         'Welcome to your redirection file');
+    } else if (type.startsWith('redirect-error')) {
+        assert.strictEqual(response.statusCode,
+            type === 'redirect-error-found' ? 302 : 301);
```

Comment:
`redirect-error-found` is in preparation for the other redirect error type for CLDSRV-489 it will be used in another PR